rpc/chain/client_utils.go

```diff
@@ -24,10 +24,10 @@ func CreateEthClientFromProvider(provider params.RpcProvider, rpcUserAgentName s
 	// Set up authentication if needed
 	switch provider.AuthType {
 	case params.BasicAuth:
-		authEncoded := base64.StdEncoding.EncodeToString([]byte(provider.AuthLogin + ":" + provider.AuthPassword))
+		authEncoded := base64.StdEncoding.EncodeToString([]byte(provider.AuthLogin.PlusString(":").Plus(provider.AuthPassword).Reveal()))
```

Comment:
How to avoid `Plus` here:
```suggestion
auth := provider.AuthLogin.Reveal() + ":" + provider.AuthPassword.Reveal()
		authEncoded := base64.StdEncoding.EncodeToString([]byte(auth))
```

Reply:
yeah, that was the old implementation, I felt not convenient when using `SensitiveString` , it should be convenient as it will be used widely, WDYT?